engine/classes/ElggDiskFilestore.php

```diff
@@ -430,4 +430,8 @@ protected function makeFileMatrix($guid) {
 		$dir = new \Elgg\EntityDirLocator($guid);
 		return $dir->getPath();
 	}
+	
+	public function moveUploadedFile($from, $to) {
+		return move_uploaded_file($from, $to);
```

Comment:
It'll need to be re-implemented for a non-local system using is_uploaded_file().